packages/web-react/src/components/FileUploader/AttachmentImagePreview.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useFileUploaderStyleProps } from './useFileUploaderStyleProps';
+
+type AttachmentImagePreviewProps = {
+  label: string;
+  imagePreview: string;
+};
+
+const AttachmentImagePreview = ({ label, imagePreview }: AttachmentImagePreviewProps) => {
+  const { classProps } = useFileUploaderStyleProps();
+
+  return (
+    <span className={classProps.attachment.image}>
+      <img src={imagePreview} alt={label} />
```

Comment:
This is my bad: we need image dimensions to prevent page jump while the image is being processed (loaded). I will fix it in the `web` package too.

```suggestion
      <img src={imagePreview} width={54} height={54} alt={label} />
```

Or maybe using a constant:

```suggestion
      <img src={imagePreview} width={IMAGE_SIZE} height={IMAGE_SIZE} alt={label} />
```

Reply:
i created constant `IMAGE_SIZE` and added width&height as you suggested